src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java

```diff
@@ -243,25 +178,21 @@ public static OfflinePlayer matchSinglePlayer(CommandSender sender, String rawUs
             // look up player according to the who is online now
             List<Player> players = Bukkit.getServer().matchPlayer(rawUsername);
             switch (players.size()) {
-                case 0:
-                    throw new CommandException("No players matched query. Use @<name> for offline lookup.");
-                case 1:
-                    return players.get(0);
-                default:
-                    throw new CommandException("More than one player found! Use @<name> for exact matching.");
```

Comment:
Did your IDE change this or did you? I feel like the indentation made sense here